### ISSUE TITLE
Ability to specify a default installer type in the manifest

### ISSUE THREAD

**Issue by mdanish-kh** [microsoft/winget-cli#3866]
### Description of the new feature / enhancement

If two or more installers are equally suitable in terms of architecture, scope and other parameters etc, there should be a way to specify a default installer type for fresh install scenarios in the manifest.

### Example

As reported in https://github.com/microsoft/winget-pkgs/issues/125309, the manifest contains both portable and nullsoft installers that are equal in terms of arch, scope etc. In fresh install flow, winget selects the installer that is defined first in the manifest. Re-ordering the manifest would make it choose the other installer.

Steps to reproduce to verify that behavior would be:

1. Locally clone winget-pkgs and install Streetwriters.Notesnook
    `winget install -m manifests/s/Streetwriters/Notesnook/2.4.0`
	
	Notice it selects the portable installer (defined earlier in the manifest)    

2. Re-order the installer nodes
3. Install the manifest locally again. Notice it will select the nullsoft installer this time.

**Comment by mdanish-kh** [comment]
[Policy] Area-Manifest

**Comment by Trenly** [comment]
I'm wondering if there shouldn't be some sort of default preference built into the client. I know it wouldn't be everyone's first choice, but I would imagine that 90% of scenarios would be covered by an order of 
* MSIX / APPX
* MSI / Wix / Burn
* EXE / Nullsoft / Inno
* Portable

**Comment by denelon** [comment]
I agree with @Trenly in terms of precedence/preference (assuming no preference/requirement exists in settings).

We have the "--installer-type" argument so a user can still get the "clean" install to be the installer type preferred, but I believe the default selection should match what was suggested.
